config/locales/en.yml

```
@@ -72,6 +72,10 @@ en:
       news_and_communications: News and communications
       statistics: Statistics
       worldwide: Worldwide
+    intervention:
+      title: Check the next steps for your limited company
+      description: You might be interested in this because you’ve been browsing guidance relevant to starting a limited company.
+      dismiss_html: <a href="?hide-intervention=true">Hide this suggestion</a> if it’s not relevant to you
```

I think this needs a `govuk-link` class to be styled correctly - it's appearing the browser default blue link at the moment.
```suggestion
      dismiss_html: <a class="govuk-link" href="?hide-intervention=true">Hide this suggestion</a> if it’s not relevant to you
```